server/mail.js

```diff
@@ -4,7 +4,7 @@ require("dotenv").config();
 const transporter = nodemailer.createTransport({
 	host: process.env.MAIL_HOST,
 	port: process.env.MAIL_PORT,
-	secure: true,
+	secure: false,
```

Comment:
Why did you change this?

Reply:
It gives a server error when you set it to true.